### DISCUSSION TITLE
how to get fast-check error reports when using jasmine with expect?

### DISCUSSION THREAD

**Discussion by vilanelle** [dubzzz/fast-check#4798] [Q&A] [answered]
Hi all :wave: 

I'm looking into adopting property-based testing with the use of this excellent library and run into a small issue.

Our test runner is karma / jasmine. When using `fast-check` with jasmine `expect` (configured as per [the docs example](https://github.com/dubzzz/fast-check-examples/blob/main/test-jasmine/example.spec.js)), the tests work fine, but the error reports are jasmine, not `fast-check`. So we miss all the valuable information from the test failure like the number of runs, property shrinking or counterexamples. 

I can use `return` with `expect().nothing()` as a workaround, but this is not ideal.

Can the library be configured to use `expect` and still get `fast-check` reports? 

Thanks!

**Comment by dubzzz** [comment]
Have you put the `stopSpecOnExpectationFailure` in your configuration of Jasmine? If you cannot, I'd recommend you to use `assert` instead of `expect` (without that flag `expect` will not throw and it's a per-requisite to make fast-check able to catch problems)

**Reply by dubzzz** [reply]
If the suggestion does not work and if you have a minimal repro of your problem, I'd happy to help.

**Reply by dubzzz** [reply, marked as answer]
I have not played with https://github.com/dubzzz/fast-check-examples/blob/main/test-jasmine/example.spec.js for a while, is it working as you expect in case of failure? If not it could be a minimal repro for me.

**Reply by vilanelle** [reply]
Hi Nicolas, many thanks for your quick response :bow:

I've tested your jasmine example and it works perfectly fine (produces proper `fast-check` logs).

It appears that our issue is tied to our setup: we're using `karma` rather than `jest`/`mocha`. So probably it's a matter of figuring out proper configs and possibly plugins.
